pkg/config/config.go

```diff
@@ -12,10 +12,10 @@ import (
 	retry "github.com/avast/retry-go"
 	"github.com/imdario/mergo"
 	"github.com/itchyny/gojq"
+	"github.com/kairos-io/kairos-sdk/bundles"
+	"github.com/kairos-io/kairos-sdk/unstructured"
 	schema "github.com/kairos-io/kairos/pkg/config/schemas"
```

Comment:
wondering, shall we make the config part of the sdk? in any case doesn't matter with this PR

Reply:
yes, indeed, it would be the perfect part.